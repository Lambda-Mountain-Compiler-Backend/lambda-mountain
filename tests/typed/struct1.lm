
type A (A( S S ));

A( 1 3 );
