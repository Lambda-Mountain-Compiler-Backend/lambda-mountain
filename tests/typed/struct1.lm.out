((() (Type ((Literal A) (App ((Literal A) (App ((Literal S) (Literal S)))))))) (GExpr (: ((App ((: ((Literal A) (And ((-> ((Cons (S S)) (A (Cons (S S))))) Atom)))) (: ((App ((: ((Literal 1) Atom)) (: ((Literal 3) Atom)))) (Cons (Atom Atom)))))) (A (Cons (S S)))))))
